commit-msys2.sh

```diff
@@ -91,7 +91,7 @@ summarize_commit () {
 		git show "$1" --format=%H \
 			-M15 --raw -- var/lib/pacman/local/\*/desc
 	fi |
-	sed -ne '/.* M\tvar\/lib\/pacman\/local\/git-extra-[1-9].*\/desc$/d' \
+	sed -ne '/.* M\tvar\/lib\/pacman\/local\/mingw-w64-*-git-extra-[1-9].*\/desc$/d' \
```

Comment:
I think we need a `.` before the `*` (it's a regular expression, not a shell pattern).

Also, maybe we should be lenient and still accept old package name _in addition_ to the new one?

```suggestion
	sed -ne '/.* M\tvar\/lib\/pacman\/local\/\(mingw-w64-.*-\)\?git-extra-[1-9].*\/desc$/d' \
```